lib/src/command/outdated.dart

```diff
@@ -325,7 +325,8 @@ Future<void> _outputHuman(List<_PackageDetails> rows,
 
   if (notAtResolvable == 0 && upgradable == 0 && rows.isNotEmpty) {
     log.message(
-        '\nDependencies are all on the latest mutually compatible versions');
+        '\nDependencies are all on the latest mutually compatible versions.'
```

Comment:
Maybe s/Dependencies are all on the latest mutually compatible versions/Dependencies are all on the latest resolvable versions/ given we have the phrase 'mutually compatible below'?

Reply:
Done